File: lab3-2/src/shoeTypes/SnowBoots.java
package shoeTypes;

import enums.Brand;
import enums.Size;
import enums.Type;
import shoe.Shoe;

public class SnowBoots extends Shoe{
    private double height;

    public SnowBoots(String name, Type type, Brand brand, Size size, String color, double price, String material, double height) {
        super(name, type, brand, size, color, price, material);
        this.height = height;
    }

    public double getHeight() {
        return height;
    }

    public void setHeight(double height) {
        this.height = height;
    }
}

File: lab14/shoes/SnowBoots.py
from shoes.Shoe import *
from enums.Type import Type


class SnowBoots(Shoe):
    shoes_type = Type.WINTER

    def __init__(self, name, id, brand, size, price, colour, shoes_colour, shoes_type):
        super().__init__(id, name, brand, price, size, shoes_colour, shoes_type)
        self.id = id
        self.name = name
        self.brand = brand
        self.size = size
        self.price = price
        self.colour = colour

    def __str__(self):
        return "Name: " + self.name + " Brand " + str(self.brand.name) + " Size " + str(self.size) + "  colour: " + str(
            self.colour) + " Type: " + str(self.type.name) + " price " + str(self.price)
